packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -705,6 +705,40 @@ const conf = (module.exports = convict({
       format: String,
     },
   },
+  surveys: {
+    enabled: {
+      default: true,
+      env: 'SURVEYS_ENABLE',
+      doc: 'Enable integrated surveys feature',
+      format: Boolean,
+    },
+    doNotBotherSpan: {
+      default: 2592000000,
+      env: 'SURVEY_DONT_BOTHER_MS',
+      doc: 'Minimum time span in milliseconds between surveys for a user',
+      format: 'integer',
+    },
+    surveys: {
+      default: [
+        // Example survey:
+        // {
+        //   id: 'portugese-speaking-mobile-users-in-southern-hemisphere',
+        //   conditions: {
+        //     platform: 'mobile',
+        //     region: 'southernHemisphere',
+        //     lang: 'pt',
+        //     relier: 'email',
+        //   },
+        //   view: 'settings',
+        //   rate: 0.1,
+        //   url: 'https://www.surveygizmo.com/s3/5541940/pizza',
+        // },
+      ],
+      doc: 'list of active surveys',
+      env: 'SURVEYS_LIST',
+      format: Array,
```

Comment:
This needs to be updated per convo below.

Reply:
I tried changing this value to `Object` and it was erroring the server. I'm not sure I understand the change that is being requested

Reply:
What was the error?  I think you need to change that default to a JSON string.  Was that the issue?  Convict trying to parse the default as a JSON string?

Reply:
I've filed FXA-2002 to work on this.